airgun/views/host.py

```diff
@@ -232,6 +239,14 @@ class ansible_roles(SatTab):
 
         resources = MultiSelect(id='ms-host_ansible_role_ids')
 
+    @View.nested
+    class virtual_machine(SatTab):
```

Comment:
We can use the compute resource profile view as it's the same for each compute resource type, this will need some code code refactoring in compute resource with moving the VMs views to a base class to compute resource profile, this will allow to use that VM views in host , compute resource profile and  profile entities.

Reply:
The class concerned is ResourceProviderProfileView